src/main/java/com/arpnetworking/metrics/mad/Main.java

```diff
@@ -250,14 +250,15 @@ private Injector launchGuice(final ActorSystem actorSystem) {
         }
 
         // Instantiate the metrics factory
+        final String sinkHost = "0.0.0.0".equals(_configuration.getHttpHost()) ? "localhost" : _configuration.getHttpHost();
```

Comment:
Nope.  Localhost was quite specific.  127.0.0.0/8 is the local range.  I've seen 127.0.1.1 be bound as well.  I think it's friendlier to use "localhost" and let the owner of the box delegate which IP address it uses.